### ISSUE TITLE
text placement in uiTextButton for Jvm is wrong

### ISSUE THREAD

**Issue by rishavs** [korlibs/korge#332]
In JVm the text in the uiTextButton is being placed way too high.

Here is what I get in the jvm target.
![image](https://user-images.githubusercontent.com/1210239/105673811-98a30980-5f0c-11eb-86e3-405988b46545.png)

Here is the same rendering using the native target
![image](https://user-images.githubusercontent.com/1210239/105673829-9e005400-5f0c-11eb-9836-6d8ec8afcccb.png)

This is the sample code;
```
import com.soywiz.korge.input.onClick
import com.soywiz.korge.scene.Scene
import com.soywiz.korge.ui.uiTextButton
import com.soywiz.korge.view.Container
import com.soywiz.korge.view.position
import com.soywiz.korge.view.text
import com.soywiz.korim.color.Colors

class MenuScene(
) : Scene() {
    suspend override fun Container.sceneInit() {
        // @TODO: Scene initialization here

        // set a background color
        views.clearColor = Colors.BLACK

        // Add a text to show the name of the game
        var gameNameText = text("Super Pong Bros II") {
            position(views.virtualWidth / 2 - 128, views.virtualHeight / 2 - 128)
        }

        var playButton = uiTextButton(256.0, 32.0) {
            text = "Play"
            position(views.virtualWidth / 2 - 128, views.virtualHeight / 2 - 64)
            onClick {
                sceneContainer.changeTo<PlayScene>()
            }
        }
        var exitButton = uiTextButton(256.0, 32.0) {
            text = "Exit"
            position(views.virtualWidth / 2 - 128, views.virtualHeight / 2)
            onClick {
                views.gameWindow.close()
            }
        }
    }
}

```

**Comment by rishavs** [comment]
Also, i just noticed that on Native the text is rather blurred.

**Comment by soywiz** [comment]
There is a known issue with text alignment when text is trying to render based on the scale on the screen. I plan to fix it this week

**Comment by soywiz** [comment]
This was fixed in 2.0.7
